### DISCUSSION TITLE
群晖 cloud sync 下  阿里云  下载失败提示  权限不足  天翼云盘正常

### DISCUSSION THREAD

**Discussion by ljj23h** [AlistGo/alist#1208] [Q&A]
![Snipaste_2022-06-08_16-49-30](https://user-images.githubusercontent.com/12965963/172574233-a98ca855-5896-4da0-951b-e903b1527c12.png)

**Comment by cbnui** [comment]
将webdav帐号更改成root一般即可

**Reply by ljj23h** [reply]
还是一样的，换成root账号了    同样的   天翼云盘没问题，应该不是webdav的账号问题

**Comment by xhofe** [comment]
本地nas运行建议打开`webdav_direct`，兼容性会好很多。
